trestlebot/entrypoints/templates/github/trestlebot-rules-transform.yml

```diff
@@ -0,0 +1,51 @@
+---
```

Comment:
Take a look at this updates version -> https://github.com/jpower432/trestlebot-demo-cd/blob/main/.github/workflows/ci.yml